deploy/crds/infra.watch_servicetelemetrys_crd.yaml

```diff
@@ -172,44 +172,62 @@ spec:
                           enabled:
                             description: Enable ElasticSearch as a storage backend for events
                             type: boolean
+                          extHostUrl:
+                            description: URL of Elasticsearch HTTP(S) endpoint
+                            type: string
+                          extTlsServerName:
+                            description: (if required) Server Name expected to match the certificate presented by the endpoint
+                            type: string
+                          extTlsSecretName:
+                            description: (if required) Name of the secret that stores the CA cert and client cert/key
+                            type: string
+                          extUserSecretName:
+                            description: (if required) Name of the secret that stores the Basic Auth credentials
+                            type: string
+                          extUseBasicAuth:
+                            description: Whether to provide HTTP Basic Auth headers
+                            type: boolean
+                          extUseTls:
+                            description: Whether to enable TLS
+                            type: boolean
                           version:
-                            description: Version of ElasticSearch to deploy. Elasticsearch licensing has changed as of version 7.11. See https://www.elastic.co/pricing/faq/licensing for details.
+                            description: [DEPRECATED - Use ext* after STF 1.5.3] Version of ElasticSearch to deploy. Elasticsearch licensing has changed as of version 7.11. See https://www.elastic.co/pricing/faq/licensing for details.
```

Comment:
I know we have a bunch of ElasticSearch items here, but it should actually be Elasticsearch. I'm not overly worried about it. I'm not sure if the branding changed at some point or I just always did this wrong in the past.